### PR TITLE
Upgrade Rails from 4.2 to Rails 5.0

### PR DESCRIPTION
**Pull request by mauro-oto**
This PR adds an article with the Rails 4.2 to 5.0 upgrade guide.

### CODE REVIEW

File: _posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+then you'll need to add:
+
+`gem 'test-unit'`
+
+to your `Gemfile`.
+
+<h2 id="gems">2. Gems</h2>
+
+- It's recommended that you check your `Gemfile` against [Ready4Rails](http://www.ready4rails.net)
+to check whether you have any older gems which may not be compatible with Rails
```
Comment (etagwerker) [inline _posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown]: @mauro-oto usually the form is "whether <a> or <b>" so I think this sounds better like "to check if you have any old gems which may ... "

File: _posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+5 yet. (As of the release of this blog post, there are only a few gems which
+don't support Rails 5 yet, this is more of a problem when you're upgrading
+early on).
+
+If any of the gems are missing on Ready4Rails, you'll need to check the Github
+page for the project manually. In case you own the gem, you'll need to make sure
```
Comment (etagwerker) [inline _posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown]: @mauro-oto I think it sounds better if you saw "you will need to check the Github page for the project one by one" or "you will need to manually check the Github page for the project"

File: _posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+parameters before attempting to upgrade to Rails 5.0.
+[`protected_attributes`](https://github.com/rails/protected_attributes) is
+no longer supported on Rails 5.0.
+
+A few months ago we worked on a project to attempt to automate this strong
+parameters migration process with the gem [RailsUpgrader](https://github.com/fastruby/rails_upgrader).
+
+There are still efforts being made to keep `protected_attributes` alive though,
+like the [`protected_attributes_continued`](https://github.com/westonganger/protected_attributes_continued)
+gem, but I would strongly recommend against using it.
```
Comment (etagwerker) [inline _posts/2018-02-02-upgrade-rails-from-4-2-to-5-0.markdown]: @mauro-oto Maybe add your reason behind this recommendation?

**Comment by mauro-oto** [comment]
@etagwerker thanks! just addressed your comments.

**Pull request merged**